### PR TITLE
Change logos to the “bridge”

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
Couldn't figure out how to change the little tab icon but I also am not sure that we need to. (certainly not for a demo)

### CODE REVIEW

File: src/assets/logo_open_bridge_large.svg
Comment (Erin-Mounts) [inline src/assets/logo_open_bridge_large.svg:1]: I noticed Stockard used different fonts in the PDF and SVG versions of the "horizontal" logo. Mike used the sans-serif one from the PDF in his presentation deck. We should probably settle on one or the other and be consistent....

File: src/assets/logo_open_bridge_small.svg
Comment (Erin-Mounts) [inline src/assets/logo_open_bridge_small.svg:1]: See other comment regarding font

**Review by Erin-Mounts — APPROVED** [review]
Looks good, with a caveat/comment about the font in the horizontal logo

**Review by hallieswan — APPROVED** [review]
Looks good! To update the tab icon (and other shortcut icons), I think we need to update the icon files listed in [public/manifest.json](https://stackoverflow.com/a/45187030), as described in [this article](https://noaheakin.medium.com/changing-the-default-react-browser-tab-title-and-icon-1240239d92d3) -- 

- public/favicon-16x16.png
- public/favicon-32x32.png
- public/favicon.png
- public/logo192.png
- public/logo512.png

**Comment by syoung-smallwisdom** [comment]
> Looks good! To update the tab icon (and other shortcut icons), I think we need to update the icon files listed in [public/manifest.json](https://stackoverflow.com/a/45187030), as described in [this article](https://noaheakin.medium.com/changing-the-default-react-browser-tab-title-and-icon-1240239d92d3) --
> 
> * public/favicon-16x16.png
> * public/favicon-32x32.png
> * public/favicon.png
> * public/logo192.png
> * public/logo512.png

Yup, Erin had the right of it. I also needed to clear my cache. Thanks!

**Pull request merged**